src/action.c

```diff
@@ -1287,6 +1342,16 @@ actions_run(struct view *activator, struct server *server,
 		case ACTION_TYPE_ZOOM_OUT:
 			magnify_set_scale(server, MAGNIFY_DECREASE);
 			break;
+		case ACTION_TYPE_WARP_CURSOR:
+		{
```

Comment:
I guess the open bracket should be at the end of the previous line. We should document that in `CONTRIBUTING.md`.

```suggestion
		case ACTION_TYPE_WARP_CURSOR: {
```

Reply:
Oh, sorry I didn't realize there's already `case ACTION_TYPE_VIRTUAL_OUTPUT_REMOVE` that follows your indentation.

Fair enough. Let's keep it as it is.